From b067134a866f902b685539add5976f807252d93f Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Mon, 23 Jan 2012 22:13:15 -0500
Subject: [PATCH] build: Don't make all target depend on check

This breaks parallel make, because the 'check' target depends on the
update-mime-database binary, but that's also listed in bin_PROGRAMS,
and automake's generated rules have $(EXEEXT) on them, so what happens
is make uses the implicit rule.

And really, make check should just be a separate target.
---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 9c2c36d..0c96439 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -68,7 +68,7 @@ uninstall-hook:
 	rm -f "$(DESTDIR)$(datadir)/mime/types"
 	rm -f "$(DESTDIR)$(datadir)/mime/version"
 
-all: check create-pot
+all: create-pot
 
 create-pot:
 	$(AM_V_GEN) $(MAKE) -C po shared-mime-info.pot
-- 
1.7.6.5

